Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1098,6 +1098,16 @@ a fixed cookie value as a trade-off. This makes all applications on the host to
 select the same service endpoint for a given service with session affinity configured.
 To disable the feature, set ``config.sessionAffinity=false``.
 
+Configure kube-proxy replacement health check server
+****************************************************
+If kube-proxy replacement is enabled and running in one of the hybrid modes mentioned
+above, the health check server address for kube-proxy should be configured to create
+HTTP endpoint to monitor the health status.
+For IPv4 interfaces set ``kube-proxy-replacement-healthz-bind-address: "0.0.0.0:10256"``
+For IPv6 interfaces set ``kube-proxy-replacement-healthz-bind-address: "[::]:10256"``
+To disable set it empty. ``kube-proxy-replacement-healthz-bind-address: ""``
+By default it is disabled.
+
```

Comment:
```suggestion
kube-proxy replacement health check server
****************************************************
To enable health check server for the kube-proxy replacement, the ``kubeProxyReplacementHealthzBindAddr`` option has to be set (disabled by default). The option accepts the IP address with port for the health check server to serve on. E.g. to enable for IPv4 interfaces set ``kubeProxyReplacementHealthzBindAddr='0.0.0.0:10256'``, for IPv6 - ``kubeProxyReplacementHealthzBindAddr='[::]:10256'``. The health check server is accessible via the HTTP ``/healthz`` endpoint.
```

Reply:
> 

Thanks will update the file and will push a new version.

Reply:
Addressed review comments.